### PR TITLE
flexbox layout, dark & nord themes, pl locales update

### PR DESCRIPTION
**Pull request by bbeny123**

### CODE REVIEW

File: src/popup.html [hunk 1]
```diff
@@ -32,35 +32,37 @@
     <div id="feed-empty"></div>
 </div>
 <div id="loading">
-    <img src="/images/loading.gif" alt="loading...">
```
Comment (olsh) [inline src/popup.html:35]: Could you please remove the old image if it's not used anymore?

File: src/scripts/popup.js [hunk 2]
```diff
@@ -129,7 +130,7 @@ $("#popup-content").on("click", ".show-content", function () {
 
                 contentContainer.html(Mustache.render(template, feed));
 		if (popupGlobal.isSidebar)
-		    contentContainer.css({width: "95%", marginLeft: 5});
+		   // contentContainer.css({width: "95%", marginLeft: 5});
```
[the remark this answers is not in the document]
Comment (bbeny123) [reply src/scripts/popup.js:133]: Yeah, I checked on firefox and chrome

[the remark this answers is not in the document]
Comment (olsh) [reply src/scripts/popup.js:133]: Please note that this is a hack for Firefox sidebar panel.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Sidebars

But if you are sure, could you please remove it?

[the remark this answers is not in the document]
Comment (bbeny123) [reply src/scripts/popup.js]: Ohh, I didn't even know about the sidebar feature 😄 
I checked it out, the hackfix is no longer necessary. But I found another bug with the sidebar, which I've already fixed,

**Comment by olsh** [comment]
Thank you for your work. I'll test this soon and merge. 👍

**Comment by olsh** [comment]
The new themes look very nice! Thank you! 👍

**Pull request merged**